supported/osg-htc/osg-hosted-ce/values.yaml

```diff
@@ -154,10 +154,11 @@ Persistence:
   LibCondorCeVolume: null
 
 # Options to allow override of the bosco directory from arbitrary git repos
-# Bosco override dirs are expected in the following location in the git repo:
+# Bosco override dirs default to the following location in the git repo:
 #   <RESOURCE NAME>/bosco_override/
 BoscoOverrides:
   Enabled: false
+  BoscoOverrideDir: null # Lets you set what directory you want to put your bosco overrides in. Defaults to Topology Resource Name value
```

Comment:
Minor nitpick: could you move this comment to its own line above like the rest of the file? It's nicer on line length and if operators copy a default values file, they're less likely to blow out the helpful comment if they modify it

Reply:
Done, anything else I should change before rebase/merge? Or am I good to go?

Reply:
LGTM